File: tea_admin/tea_admin/report/kamjari_statement/kamjari_statement.py
# Copyright (c) 2013, Frappe and contributors
# For license information, please see license.txt

from __future__ import unicode_literals
import frappe
from frappe import _
from frappe.model.document import Document
from frappe import utils


def execute(filters=None):
	data = []
	columns=get_columns()
	return columns, data


def get_columns():
		
		columns = [{
			"fieldname": "date",
				"label": _("Date"),
				"fieldtype": "date",
				"option":"attendence",
				"width": 135

		}]
		columns.append({
				"fieldname": "emp_code",
				"label": _("Emp Code"),
				"fieldtype": "Data",
				"option":"attendence",
				"width": 135
	    })


		columns.append({
				"fieldname": "name1",
				"label": _("Emp Name"),
				"fieldtype": "Data",
				"option":"attendence",
				"width": 135
	    })


		

	    
		return columns

File: tea_admin/tea_admin/report/kamjari_statement/kamjari_statement.js
// Copyright (c) 2016, Frappe and contributors
// For license information, please see license.txt

frappe.query_reports["Kamjari Statement"] = {
	"filters": [
	{
            "fieldname":"date1",
            "label": __("FROM DATE"),
            "fieldtype": "Date",
            "options": "",
            "default": frappe.datetime.get_today(),
            "reqd":0
        },

    {
            "fieldname":"date2",
            "label": __(" TO DATE"),
            "fieldtype": "Date",
            "options": "",
            "default": frappe.datetime.get_today(),
            "reqd":0
        }

	]
}
